### PR TITLE
Add version service and include in layout

### PR DESCRIPTION
**Pull request by tonyjoanes**
Added service to get the current version and include this in the view.

App version is set in the layout page under the footer, like so:

image

### CODE REVIEW

File: src/Marvin.Web/Code/Bootstrap/ApplicationVersion.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+
+namespace Marvin.Web.Code.Bootstrap;
+
+/// <summary>
+/// Service providing the version of the application
+/// </summary>
+public interface IAppVersionService
```
Comment (AshenW0lf) [inline src/Marvin.Web/Code/Bootstrap/ApplicationVersion.cs:8]: Is Bootstrap the right place for this code, it feels like it should be in it's own folder with the class and interface being separate files?

Comment (tonyjoanes) [reply src/Marvin.Web/Code/Bootstrap/ApplicationVersion.cs]: @AshenW0lf moved it to the `Application` folder.

Also an approach I am trying is where code has a single use interface like this, is to keep code + interface together as they will change for the same reasons and its nice and quick to update things together like this.

Similar with the single project solution, these days we don't need to separate things out into multiple projects and its a nicer developer experience to work in a vertical architecture in a single solution where its quick and easy to find things... anyway that is the idea :-)

Comment (tonyjoanes) [reply src/Marvin.Web/Code/Bootstrap/ApplicationVersion.cs]: oh this is all similar to the `Flat Areas` approach to, keep like things together

Comment (AshenW0lf) [reply src/Marvin.Web/Code/Bootstrap/ApplicationVersion.cs]: sounds good to me. :-)

**Pull request merged**